poetry.lock

```diff
@@ -2,24 +2,24 @@
 
 [[package]]
 name = "altgraph"
-version = "0.17.3"
+version = "0.17.4"
```

Comment:
version update on that package was intended?

Reply:
All updates are according to the specifications in the pyproject.toml; just the lock file got updated and that on purpose, yeah. I think that hasn't happened in a while.